### PR TITLE
[Subtask][KPIP-5] Github CI check kyuubi web ui

### LINKED ISSUE #3335: [Subtask] [KPIP-5] Github CI add web ui check
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues) and found no similar issues.

### Describe the subtask

 Github CI add web ui check

### Parent issue

#3248 

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

For code review and check pr, we need Github CI help check web ui.
Run `npm run coverage && npm run build-check` to find syntax issues and build errors.
Actions used node from https://github.com/marketplace/actions/setup-node-js-environment

Close #3335
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by pan3793** [comment]
let's separate the eslint work to style.yml

**Comment by zwangsheng** [comment]
CC @tongwl

**Comment by zwangsheng** [comment]
Seems pass CI check, PLZ help check  whether the style check notice is appropriate.
Friendly ping @tongwl @pan3793

**Comment by zwangsheng** [comment]
Friendly ping @pan3793, plz help review this CI.

File: .github/workflows/web-ui.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      # from https://github.com/marketplace/actions/setup-node-js-environment
+      - name: setup npm
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: npm run coverage & build
+        run: >-
```
Comment (pan3793) [inline .github/workflows/web-ui.yml:30]: use `|`

File: .github/workflows/web-ui.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  cancel-in-progress: true
+
+jobs:
+  web-ui:
+    name: Kyuubi Web UI check
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      # from https://github.com/marketplace/actions/setup-node-js-environment
```
Comment (pan3793) [inline .github/workflows/web-ui.yml:24]: unnecessary comment

File: .github/workflows/style.yml [hunk 3]
```diff
@@ -58,3 +58,20 @@ jobs:
...
       - name: JavaStyle with maven
         run: build/mvn spotless:check -Pflink-provided,hive-provided,spark-provided,spark-block-cleaner,spark-3.3,spark-3.2,spark-3.1,tpcds
+      - name: setup npm
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Web UI Style with node
+        run: >-
+          cd ./kyuubi-server/web-ui &&
+          npm install &&
```
Comment (pan3793) [inline .github/workflows/style.yml:68]: use pnpm?

File: .github/workflows/style.yml [hunk 4]
```diff
@@ -58,3 +58,20 @@ jobs:
...
+          npm run lint &&
+          echo "---------------------------------------Notice------------------------------------" &&
+          echo "If this style check fail, you should follow these commands for a tentative fix" &&
+          echo "* cd ${KYUUBI_HOME}/kyuubi-server/web-ui" &&
```
Comment (pan3793) [inline .github/workflows/style.yml:72]: use relative path here

**Review by pan3793 — APPROVED** [review]

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**